piptools/repositories/pypi.py

```diff
@@ -219,8 +223,7 @@ def resolve_reqs(self, download_dir, ireq, wheel_cache):
                 if PIP_VERSION < (19, 4):
                     resolver.require_hashes = require_hashes
                     results = resolver._resolve_one(reqset, ireq)
-                else:  # pragma: no cover
-                    # TODO remove pragma after pip==19.4 being released
```

Comment:
FTR, this change addressed to https://github.com/jazzband/pip-tools/pull/953#discussion_r338850822.